src/content/glossary/index.md

```diff
@@ -840,6 +856,14 @@ The smallest denomination of [ether](#ether). 10<sup>18</sup> wei = 1 ether.
 
 A special Ethereum address, composed entirely of zeros, that is specified as the destination address of a [contract creation transaction](#contract-creation-transaction).
 
+### Zero-knowledge proof {#zk-proof}
+
+Zero-knowledge proof is a cryptographic method by which one party (the prover) can prove to another party (the verifier) that a given statement is true while the prover avoids conveying any additional information.
```

Comment:
I'd be in favour of using simpler language here.

Suggesting: 

1. I don't think we need to specify person A proves to person B. I believe the number of verifiers in a zkp could be arbitrary and we can therefore generalise this statement. (correct me if I'm wrong here)
2. Remove formal language for more colloquial language

```suggestion
Zero-knowledge proof is a cryptographic method that allows an individual to prove that a statement is true without conveying any additional information.
```

Reply:
You are right. Maybe even change "individual" for "entity". But it is understandable as proposed.